### ISSUE TITLE
Keyboard or swipe should cause idle event (hide UI as with native iPhone apps)

### ISSUE THREAD

**Issue by scottkellum** [Treesaver/treesaver#61]
I think the time-out should be faster and when keyboard navigation is being used it should hide. I am worried about the chrome covering up the pages while someone is reading.

**Comment by scottkellum** [comment]
It also times out while navigation is in use but mouse is not moving. If possible register a click on chrome to restart the timer.

**Comment by fortes** [comment]
Partial-fix -- SHA: 454e4f88dda7499dd0f54004ea999d1f61beef92

Creating a separate bug for use of Chrome not causing activation
